components/input.tsx

```diff
@@ -98,7 +98,7 @@ export default class Input extends React.PureComponent<Props, State> {
     public render() {
         const {value, placeholder, className, error: propError, hasError, addon, name, textPrefix, ...otherProps} = this.props;
         const {focused, error: stateError} = this.state;
-        let inputClass = className ? `Input ${className}` : 'Input';
+        let inputClass = className ? `Input form-control ${className}` : 'Input form-control';
```

Comment:
Is this a common component that's used everywhere in the webapp? will including `form-control` have any adverse affect on the styling for all the non-cloud cases?

Reply:
I just added this component with the rest of the cloud stuff, so it's not being used by anything but the Cloud Onboarding flow right now.

Reply:
👌